### PR TITLE
update install instruction and clean up scripts

### PR DESCRIPTION
**Pull request by richardwxn**
1. update instruction to avoid confusion
2. remove stale install from operator repo script
3. use istioctl as default for script

/cc @incfly

### CODE REVIEW

File: perf/istio-install/setup_istio.sh [hunk 1]
```diff
@@ -142,7 +143,7 @@ function install_istio_with_helm() {
 function install_istio_with_istioctl() {
   local CR_PATH="${WD}/istioctl_profiles/${CR_FILENAME}"
   pushd "${ISTIOCTL_PATH}"
-  ./istioctl manifest apply -f "${CR_PATH}" --set "${SET_OVERLAY}" "${EXTRA_ARGS}"
+  ./istioctl manifest apply -f "${CR_PATH}" --set "${SET_OVERLAY}" "${EXTRA_ARGS}" --wait --force=true
```
Comment (howardjohn) [inline perf/istio-install/setup_istio.sh]: why force?

Comment (howardjohn) [inline perf/istio-install/setup_istio.sh]: why force?

File: perf/istio-install/README.md [hunk 2]
```diff
@@ -28,53 +28,14 @@ The script will create files to be used later in the setup, as config maps:
...
-
-### Installing release candidates
-
-```bash
-DNS_DOMAIN=v11mcp.qualistio.org ./setup_istio_release.sh 1.1.4 pre-release
-```
+1. To setup Istio with prerelease candidates, run `DNS_DOMAIN=your-example-domain ./setup_istio_release.sh 1.5.0-alpha.0 pre-release`
 
-This option uses pre-release buckets to get builds and charts.
+To provide extra override via set or yaml file, run with EXTRA_ARGS, for example: `EXTRA_ARGS="--set values.grafana.enabled=true -f overlay.yaml" DNS_DOMAIN=your-example-domain ./setup_istio_release.sh 1.5.0`
```
Comment (incfly) [inline perf/istio-install/README.md]: make this as part of list items , adding `1.` in the beginning as well?

File: perf/istio-install/README.md [hunk 3]
```diff
@@ -28,53 +28,14 @@ The script will create files to be used later in the setup, as config maps:
...
-To overwrite helm flags, create a file to hold helm flags you want to overwrite and save as extra-values.yaml or other file names.
-
-```bash
-DNS_DOMAIN=your-example-domain EXTRA_VALUES=extra-values.yaml ./setup_istio.sh release-1.1-20190203-09-16
-```
-
-### Installing release candidates
-
-```bash
-DNS_DOMAIN=v11mcp.qualistio.org ./setup_istio_release.sh 1.1.4 pre-release
```
Comment (incfly) [inline perf/istio-install/README.md]: setup_istio_releaess.sh also support specific SHA as well right? could you add an example here?

export ISTIO_RELEASE=1.6-alpha.0ef2cd46e2da64b9252c36ca4bf90aa474b73610, something like that.

Comment (howardjohn) [reply perf/istio-install/README.md]: fwiw, I have a script I have started on locally but haven't gotten around to merging, that supports install of any release:

```


WD=$(dirname "$0")
WD=$(cd "$WD"; pwd)
DIRNAME="${WD}/tmp"
mkdir -p "${DIRNAME}"
export GO111MODULE=on

# Passing a tag, like latest or 1.4-dev
if [[ -n "${TAG:-}" ]]; then
  VERSION=$(curl -sL https://gcsweb.istio.io/gcs/istio-build/dev/"${TAG}")
  OUT_FILE="istio-${VERSION}"
  RELEASE_URL="https://storage.googleapis.com/istio-build/dev/${VERSION}/istio-${VERSION}-linux.tar.gz"
# Passing a dev version, like 1.4-alpha.41dee99277dbed4bfb3174dd0448ea941cf117fd
elif [[ -n "${DEV_VERSION:-}" ]]; then
  OUT_FILE="istio-${DEV_VERSION}"
  RELEASE_URL="https://storage.googleapis.com/istio-build/dev/${DEV_VERSION}/istio-${DEV_VERSION}-linux.tar.gz"
# Passing a version, like 1.4.2
elif [[ -n "${VERSION:-}" ]]; then
  OUT_FILE="istio-${VERSION}"
  RELEASE_URL="https://storage.googleapis.com/istio-prerelease/prerelease/${VERSION}/istio-${VERSION}-linux.tar.gz"
# Passing a release url, like https://storage.googleapis.com/istio-prerelease/prerelease/1.4.1/istio-1.4.1-linux.tar.gz
elif [[ -n "${RELEASE_URL:-}" ]]; then
  OUT_FILE=${OUT_FILE:-"$(basename "${RELEASE_URL}" -linux.tar.gz)"}
# Passing a gcs url, like gs://istio-build/dev/1.4-alpha.41dee99277dbed4bfb3174dd0448ea941cf117fd
elif [[ -n "${GCS_URL:-}" ]]; then
  DOWNLOAD_TYPE=gcs
  RELEASE_URL="${GCS_URL}"
  OUT_FILE=${OUT_FILE:-"$(basename "${RELEASE_URL}" -linux.tar.gz)"}
fi

if [[ -z "${RELEASE_URL:-}" ]]; then
  echo "Must set on of TAG, VERSION, DEV_VERSION, RELEASE_URL, GCS_URL"
  exit 2
fi

function download_release() {
  outfile="${DIRNAME}/${OUT_FILE}"
  if [[ ! -d "${outfile}" ]]; then
    tmp=$(mktemp -d)
    if [[ "${DOWNLOAD_TYPE:-}" == gcs ]]; then
      gsutil cp "${GCS_URL}" "${tmp}/out.tar.gz"
      tar xvf "${tmp}/out.tar.gz" -C "${DIRNAME}"
    else
      curl -fJLs -o "${tmp}/out.tar.gz" "${RELEASE_URL}"
      tar xvf "${tmp}/out.tar.gz" -C "${DIRNAME}"
    fi
  else
    echo "${outfile} already exists, skipping download"
  fi
}

function install_istioctl() {
  release=${1:?release folder}
  shift
  "${release}/bin/istioctl" manifest apply --skip-confirmation --wait "${@}"
}

function install_gateways() {
  helm template --set domain="${domain}" base | kubectl -n istio-system apply -f -
}

function install_prom_op() {
  "$WD/setup_prometheus.sh" "${DIRNAME}"
}

download_release
install_istioctl "${DIRNAME}/${OUT_FILE}" "${@}"

install_prom_op
install_gateways
```

File: perf/istio-install/setup_istio.sh (hunk 1 above)
Comment (richardwxn) [reply perf/istio-install/setup_istio.sh]: unnecessary here, removed

File: perf/istio-install/README.md (hunk 2 above)
Comment (richardwxn) [reply perf/istio-install/README.md]: done

**Comment by carolynhu** [comment]
@richardwxn are you going to finish this PR or it is stale.

**Pull request closed without merging**